deployment/conf/.templates/start_server.sh.templ

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+
+JAVA_OPTS="-Djava.awt.headless=true -server \
+               -Xms{{ default .Env.min_memory "1000" }}m -Xmx{{ default .Env.max_memory "3000" }}m \
+               -XX:+UseG1GC"
+
+java -DSTOP.PORT=8079 -DSTOP.KEY=foo -Djetty.home=$JETTY_HOME -jar $JETTY_HOME/start.jar
```

Comment:
I can't figure out where JETTY_HOME is being set

Reply:
The kbase/kb_jre base image uses the jetty:jre8 image as the ultimate base, which is a fully configured jetty9 base image. The details of the base image are here:
https://hub.docker.com/_/jetty/

Reply:
Can you document this in a comment? Too much magic for me.

Reply:
Docs entered